lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -22,7 +22,7 @@ def call(package)
             return [ValidationError.new(:missing_manifest)]
           end
 
-          collate_manifest_errors(package)
+          collate_manifest_errors(package, opts.fetch(:skip_password_parameter_check, true) )
```

Comment:
Maybe I've been a manager too long, but I'm finding this a bit hard to follow. We default the `skip_password_parameter` value to false [here](https://github.com/zendesk/zendesk_apps_support/pull/364/files#diff-3bda7d468632d3fefde0a7a1b79ff0e590a24cdeaa753db9fa28493d6f6c2a5eR34), then default it to true [here](https://github.com/zendesk/zendesk_apps_support/pull/364/files#diff-721c968da45bad1e120fc0e48af8c3a66d2815bf105dc3ca915c48d6f823ddceR25) then apply the rule conditionally with a double negative.

Would it be simpler to change the variable name to `apply_parameter_check` and just give it the appropriate default value in one place?

Reply:
yeah that would be more readable. I have updated it now.

Reply:
Could we get the PR description updated with the new implementation using `apply_parameter_check` please?